code/datums/ai/hunting_behavior/hunting_behaviors.dm

```diff
@@ -139,6 +139,10 @@
 /datum/ai_behavior/hunt_target/unarmed_attack_target/reset_target
 	always_reset_target = TRUE
 
+/datum/ai_behavior/hunt_target/unarmed_attack_target/reset_target_combat_mode
+	always_reset_target = TRUE
+	switch_combat_mode = TRUE
```

Comment:
Also worth keeping an eye on, if we get too many of these var subtypes they need to move into the blackboard.